### PR TITLE
fixes https://github.com/GoogleWebComponents/google-map/issues/275

### PR DESCRIPTION
**Pull request by eximius313**

### CODE REVIEW

File: google-map.html [hunk 1]
```diff
@@ -243,6 +243,11 @@
         observer: '_disableDefaultUiChanged'
       },
 
+      disableMapTypeControl: {
```
Comment (ebidel) [inline google-map.html]: This needs a doc string and the indentation looks off.

File: google-map.html [hunk 2]
```diff
@@ -684,6 +690,13 @@
       this.map.setOptions({disableDefaultUI: this.disableDefaultUi});
     },
 
+    _disableMapTypeControlChanged: function() {
+        if (!this.map) {
```
Comment (ebidel) [inline google-map.html]: 2 space indent

**Comment by ebidel** [comment]
Thanks for this. Do you mind adding a test?

**Comment by eximius313** [comment]
@ebidel I fixed things you have commented.
Unfortunately I have no idea how to add theese test's and I don't have much time to learn it right now :(

File: google-map.html [hunk 3]
```diff
@@ -244,6 +244,14 @@
       },
 
       /**
+       * If set, removes the map's 'map type' UI controls.
+       */
+      disableMapTypeControl: {
+          type: Boolean,
```
Comment (ebidel) [inline google-map.html]: 2-space indent here

Comment (eximius313) [inline google-map.html]: fixed

**Comment by ebidel** [comment]
LGTM

**Pull request merged**